eg-next/examples/primitives-fill.rs

```diff
@@ -33,21 +33,21 @@ where
         .into_styled(style)
         .draw(target)?;
 
-    Ellipse::new(Point::new(96 * 3, 0), ELLIPSE_SIZE)
+    Ellipse::new(Point::new(0 + 24, 97), ELLIPSE_SIZE)
         .into_styled(style)
         .draw(target)?;
 
     RoundedRectangle::new(
         Rectangle::new(Point::new(32, 0), Size::new(64, 64)),
         CornerRadii::new(Size::new(16, 16)),
     )
-    .translate(Point::new(96 * 4, 0))
+    .translate(Point::new(96 + 24, 97))
     .into_styled(style)
     .draw(target)
 }
 
 fn main() -> Result<(), Infallible> {
-    let mut display: SimulatorDisplay<BinaryColor> = SimulatorDisplay::new(Size::new(512, 128));
+    let mut display: SimulatorDisplay<BinaryColor> = SimulatorDisplay::new(Size::new(300, 200));
 
     let stroke = PrimitiveStyle::with_stroke(BinaryColor::On, 1);
 
```

Comment:
The `stroke_off_fill_off` style behaves the same as a `PrimitiveStyle::with_fill(BinaryColor::Off)`. Would it be better to change the example to use colors to better differentiate the styles?